CHANGES.md

```diff
@@ -240,19 +240,20 @@ UUID from the `linkerd-config` ConfigMap!
   * The `stat` command now always shows the number of open TCP connections
   * **Breaking Change** Removed the `--disable-external-profiles` flag from the
     `install` command; external profiles are now disabled by default and can be
-    enabled with the new `--enable-external-profiles` flag
+     enabled with the new `--enable-external-profiles` flag
 * Controller
   * The auto-inject admission controller webhook is updated to watch pods
-    creation and update events; with this change, proxy auto-injection now works
-    for all kinds of workloads, including StatefulSets, DaemonSets, Jobs, etc
+    creation and update events; with this change, proxy auto-injection now
+    works for all kinds of workloads, including StatefulSets, DaemonSets,
+    Jobs, etc
 * Proxy
   * Some `l5d-*` informational headers have been temporarily removed from
     requests and responses because they could leak information to external
     clients
 * Web UI
   * The topology graph now shows TCP stats if no HTTP stats are available
-  * Improved table display on the resource detail page for resources with
-    TCP-only traffic
+  * The resource detail page no longer shows blank tables if the resource only
```

Comment:
intentional rewrite? if so, consider rewording same line-item in the 2.3.0 section

Reply:
Hmm, I didn't rewrite this prior to the merge conflict, thanks for catching that.

Reply:
Must have been during the merge conflict